main.go

```diff
@@ -106,7 +106,9 @@ func main() {
 
 	version.PrintVersionLogs(setupLog)
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), config.ManagerOptionsWithNamespaces(setupLog, ctrl.Options{
+	restConfig := ctrl.GetConfigOrDie()
+	restConfig.UserAgent = "eds-controller-manager"
```

Comment:
I'm open to suggestions for the new manager name. Perhaps `eds-controller` is sufficient?

Reply:
IMO `eds-controller` is sufficient

Reply:
Thanks! Updated